docs/changes/timeseries/17351.bugfix.rst

```diff
@@ -0,0 +1 @@
+Made ``TimeSeries`` and ``BinnedTimeSeries`` more robust to subclassing.
```

Comment:
Why don't you list all the methods that are fixed? There's only three of them.

Reply:
no reason. Updated